sklearn/ensemble/_forest.py

```diff
@@ -1144,9 +1134,19 @@ class labels (multi-output problem).
 
     .. [1] L. Breiman, "Random Forests", Machine Learning, 45(1), 5-32, 2001.
 
-    See Also
+    Examples
     --------
-    DecisionTreeClassifier, ExtraTreesClassifier
+    >>> from sklearn.ensemble import RandomForestClassifier
+    >>> from sklearn.datasets import make_classification
+
```

Comment:
Could you remove this blank line. It will make 2 different blocks in the HTML rendering.